Etabs_Adapter/CRUD/Create.cs

```diff
@@ -71,6 +71,10 @@ protected override bool Create<T>(IEnumerable<T> objects, bool replaceAll = fals
                 List<Diaphragm> diaphragms = panels.Select(x => x.Diaphragm()).Where(x => x != null).ToList();
 
                 this.Replace(diaphragms);
+
+                List<Pier> piers = panels.Select(x => x.Pier()).Where(x => x != null).ToList();
+
+                this.Replace(piers);
```

Comment:
you are right, this is probably because 1/2 way through i changed my mind as to what the pier and spandrels actually are.  The way i see it, the pier and spandrels are actually just labels to panels.  therefore, to create a pier or spandrel isn't actually something that ever happens, but rather, you update append info to a panel.  for this project, we already had an etabs model which needed amending so thats why the update procedure was implemented only.